otx/algorithms/action/configs/classification/configuration.yaml

```diff
@@ -198,6 +198,23 @@ learning_parameters:
       type: UI_RULES
     visible_in_ui: true
     warning: This will automatically control the patience and interval when early stopping is enabled.
+  auto_decrease_batch_size:
+    affects_outcome_of: TRAINING
+    default_value: false
+    description: Find a proper batch size by training for an iteration with various batch size a few times.
+    editable: true
+    header: Decrease batch size if current batch size isn't fit to CUDA memory.
+    type: BOOLEAN
+    ui_rules:
+      action: DISABLE_EDITING
+      operator: AND
+      rules: []
+      type: UI_RULES
+    visible_in_ui: true
+    warning:
+      It can decrease batch size if default or user's input batch size isn't available.
+      If batch size decreases, learning rate is also adapted depending on adapted batch size.
+      Because it needs additional training to check, it spends a little bit extra time.
```

Comment:
```suggestion
      Enabling this option could reduce the actual batch size if the current setting results in out-of-memory error.
      The learning rate also could be adjusted according to the adapted batch size.
      This process might take some extra computation time to try a few batch size candidates.
```

Reply:
Obviously, for all other tasks, too :)

Reply:
Nice refinement. I'll update at all part :)

Reply:
updated it.